transports/sqs/configuration-options_visibility-timeout_sqs_[7.3,).partial.md

```diff
@@ -0,0 +1,87 @@
+## Message visibility
+
+To prevent messages from being reprocessed while a handler is still executing, the transport automatically renews the message visibility timeout during processing. This is especially important for long-running handlers where the processing time may exceed the original visibility timeout.
+
+The transport calculates when to renew the message visibility using the following logic:
+
+```text
+buffer = Min(remainingTime / 2, 10 seconds)
+renewAfter = remainingTime - buffer
+```
+
+Renewal is triggered after `renewAfter` has elapsed since the last renewal. If the remaining time is less than 400ms, the renewal is attempted immediately. This ensures the message visibility is extended consistently and early enough to avoid expiration, even under system load or scheduling delays.
```

Comment:
```suggestion
A message visibility timeout renewal is triggered after `renewAfter` has elapsed since the last renewal attempt. The renewal is attempted immediately if the remaining time is less than 400ms. This ensures the message visibility is extended consistently and early enough to avoid expiration, even under system load or scheduling delays.
```